lib/gollum/app.rb

```diff
@@ -22,6 +22,13 @@
 # Fix to_url
 class String
   alias :upstream_to_url :to_url
+
+  unless defined?(Grit)
```

Comment:
Please don't introduce a reference to the Grit class in gollum. How about checking which git adapter gollum is using?